documentation/rfc/line_sections.md

```diff
@@ -0,0 +1,165 @@
+The line sections is a way to impact the some routes between 2 stops areas
+
+We know (thanks to @eturk!) how to handle blocking line section (the buses don't stop anymore at the stops between the 2 stops), now we need to kown how to show them
+
+Here's a proposal on how to handle them:
+
+# representation
+
+I think the easiest way to do show them would be as a classic pt_object
+```javascript
+{
+  "status":"active",
+  // lots of additional fields
+  "impacted_objects":[
+    {
+      "pt_object":{
+        "embedded_type":"line_section", // <--- here we add a new embedded_type
+        "line_section":{
+            "from": {
+              // a stop area
+            },
+            "to": {
+              // a stop area
+            },
+            "line": {
+               // the line
+            }
+            // for the moment the idea is not to dump the associated routes
+        },
+        "quality":0,
+        "id":"id_of_the_line_section",
+        "name":"Line section"
+      }
+    }
+  ]
+  "id":"002def5c-dc76-11e6-8542-005056a47b86"
```

Comment:
For me it's not a good idea to rely on external module for uri.
But displaying chaos's id is another thing...
Pretty sure we have to discuss about communication with chaos.